ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -535,6 +557,19 @@ interface Ethernet28
 
 # Filters
 
+# DOT1X
```

Comment:
EOS documentation use the `802.1X Port Security` term. I think we should align to that here.
So
```
# 802.1X Port Security

## 802.1X Summary

### 802.1X Interfaces
```

Reply:
completed.